client/src/main/java/com/arcbees/beestore/client/application/CurrentOrderImpl.java

```diff
@@ -30,11 +29,15 @@
 import com.google.web.bindery.event.shared.EventBus;
 
 public class CurrentOrderImpl implements CurrentOrder, HasHandlers {
+    private static final float TPS = 0.09975f;
+    private static final float TVQ = 0.05f;
```

Comment:
If this ever go live, we'll need an other way to calculate the taxes. It depends on the shipping address